contrib/exporters/secadvisor/mod/acc_rej_action.go

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright (C) 2019 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package mod
+
+import (
+	"time"
+
+	"github.com/skydive-project/skydive/flow"
+	//"github.com/skydive-project/skydive/logging"
+	"github.com/spf13/viper"
+)
+
+type FlowInfoStatus int32
+
+const (
+	FlowInfo_ONE_ENTRY   FlowInfoStatus = 1
+	FlowInfo_TWO_ENTRIES FlowInfoStatus = 2
+	FlowInfo_OLD_FLOW    FlowInfoStatus = 3
```

Comment:
the standard naming convention for consts is CamelCase:

```
const (
         FlowInfoOneEntry FlowInfoStatus = 1
         ...
)
```

Reply:
OK